<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWPosSel</class>
 <widget class="QDialog" name="CWPosSel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select world location</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1,1,1">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Location</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Longitude</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0" colspan="3">
                <widget class="QDoubleSpinBox" name="doubleSpinBox">
                 <property name="suffix">
                  <string>째</string>
                 </property>
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="maximum">
                  <double>180.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBox">
                 <property name="suffix">
                  <string>째</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>180</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSpinBox" name="spinBox_3">
                 <property name="suffix">
                  <string>&quot;</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>West</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>East</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBox_2">
                 <property name="suffix">
                  <string>'</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="3">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Latitude</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="4">
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0" colspan="3">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                 <property name="suffix">
                  <string>째</string>
                 </property>
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="maximum">
                  <double>90.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>North</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="spinBox_4">
                 <property name="suffix">
                  <string>째</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>90</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBox_5">
                 <property name="suffix">
                  <string>'</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSpinBox" name="spinBox_6">
                 <property name="suffix">
                  <string>&quot;</string>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="maximum">
                  <number>59</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QRadioButton" name="radioButton_4">
                 <property name="text">
                  <string>South</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Altitude</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="1">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Altitude above sea level</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton_6">
                 <property name="toolTip">
                  <string>Get from 'Google API'</string>
                 </property>
                 <property name="text">
                  <string>Get...</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QSpinBox" name="spinBox_7">
                 <property name="suffix">
                  <string> m</string>
                 </property>
                 <property name="minimum">
                  <number>-500</number>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
              <zorder>spinBox_7</zorder>
              <zorder>label</zorder>
              <zorder>pushButton_6</zorder>
              <zorder>horizontalSpacer_4</zorder>
              <zorder>verticalSpacer_3</zorder>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Select from map...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Get location from IP</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Time zone</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="toolTip">
             <string>Get from 'Google API'</string>
            </property>
            <property name="text">
             <string>Get...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="radioButton_6">
            <property name="text">
             <string>After UTC</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Before UTC</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> h</string>
            </property>
            <property name="maximum">
             <double>24.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_11">
            <property name="toolTip">
             <string>Get time zone from system</string>
            </property>
            <property name="text">
             <string>System TZ</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>doubleSpinBox_3</zorder>
         <zorder>radioButton_5</zorder>
         <zorder>radioButton_6</zorder>
         <zorder>pushButton_7</zorder>
         <zorder>horizontalSpacer_5</zorder>
         <zorder>pushButton_11</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Daylight saving time</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
            <property name="prefix">
             <string>+</string>
            </property>
            <property name="suffix">
             <string> h</string>
            </property>
            <property name="maximum">
             <double>8.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_10">
            <property name="toolTip">
             <string>Get from 'Google API' (Current DST)</string>
            </property>
            <property name="text">
             <string>Get...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_12">
            <property name="toolTip">
             <string>Get daylight saving from system</string>
            </property>
            <property name="text">
             <string>System DST</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Atmosphere</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_9">
            <property name="suffix">
             <string> hPa</string>
            </property>
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>1500</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBox_8">
            <property name="suffix">
             <string>째 Deg</string>
            </property>
            <property name="minimum">
             <number>-200</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Atm. pressure</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Disable atm. refraction</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_tempType"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Saved locations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="CSearchWidget" name="widget" native="true"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <item>
           <widget class="CListWidget" name="listWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>New...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_16">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Update...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>Home location</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="10,1,1,1,1">
             <item>
              <widget class="QComboBox" name="cb_home"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_13">
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_14">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_15">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSearchWidget</class>
   <extends>QWidget</extends>
   <header>csearchwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CListWidget</class>
   <extends>QListWidget</extends>
   <header>clistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
